### PR TITLE
Regarding open issue #188 and #142

### PR DESCRIPTION
**Pull request by manzanillo**
Hello together,
here is a proposal for issue #188. The first commit only changes the behavior, so there is no wrong rotating in 2D mode anymore.
The other 2 commits changed the name of parallel projection to 2D mode as suggested in #188.
Best wishes
Stefan

### CODE REVIEW

**Comment by manzanillo** [comment]
Oh and sorry for the auto formatting of randomly distributed spaces O:-)

**Comment by ericrosenbaum** [comment]
Thanks @manzanillo! Please note that it may take us a while to review your contribution due to political instability in @bromagosa's part of the world, but hopefully we'll get to it soon.

File: beetleblocks/objects.js [hunk 1]
```diff
@@ -1340,20 +1340,24 @@ StageMorph.prototype.initCamera = function () {
...
-                center = boundingSphere.center,
-                distance = boundingSphere.radius;
+          this.reset();
 
-            this.reset();
+          var boundingBox = new THREE.Box3().setFromObject(myself.myObjects),
+              boundingSphere = boundingBox.getBoundingSphere(),
+              center = boundingSphere.center,
+              distance = boundingSphere.radius;
+              console.log(boundingBox)
```
Comment (bromagosa) [inline beetleblocks/objects.js]: needs to be removed

Comment (manzanillo) [reply beetleblocks/objects.js]: Shouldn't be included in the pull request, but now I changed the rest to (hopefully) fix 142.

**Comment by bromagosa** [comment]
Thank you so much, and really sorry about the delay on pulling this!

**Pull request merged**